torch/cuda/__init__.py

```diff
@@ -28,6 +28,11 @@
 from .. import device as _device
 from .._utils import _dummy_type, _LazySeedTracker, classproperty
 from ._utils import _get_device_index
+from .gds import (
```

Comment:
I don't have a super strong opinion on my end but since these are very experimental and we might change them. I think it would be ok to only have them in torch.cuda.gds for now?

cc @eqy in case you have a preference here?

Reply:
Okay, keeping them all as private in `torch.cuda._gds` since they aren't in binary builds yet